pkg/clustermesh/mcsapi/types/config.go

```diff
@@ -24,4 +28,14 @@ func (cfg MCSAPIConfig) Flags(flags *pflag.FlagSet) {
 		cfg.ClusterMeshEnableMCSAPI,
 		"Whether or not the MCS API support is enabled.",
 	)
+	flags.Bool(
+		"clustermesh-install-mcs-api-crds",
+		cfg.ClusterMeshInstallMCSAPICRDs,
+		"Whether or not the MCS API CRDs should be automatically installed. "+
```

Comment:
Nit. Prefer a more direct phrasing. E.g., "Install and manage the MCS API CRDs. Only applicable if MCS API support is enabled."